accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -1176,13 +1182,12 @@ public void onConnectionCreated(Session session, Connection connection) {
     public void onConnectionDestroyed(Session session, Connection connection) {
       LOG.d(LOG_TAG, "onConnectionDestroyed: ", connection.getData());
       mSession.connections.remove(connection.getConnectionId());
-      mConnectionsCount--;
       if (connection.getCreationTime().compareTo(mSessionConnection.getCreationTime()) <= 0) {
         mOlderThanMe--;
       }
       if (mBasicListeners != null) {
         for (BasicListener listener : mBasicListeners) {
-          ((RetriableBasicListener) listener).onDisconnected(SELF, mConnectionsCount,
+          ((RetriableBasicListener) listener).onDisconnected(SELF, mSubscribers.size(),
```

Comment:
I don't think we should replace the number of connections by number of subscribers